build.gradle

```diff
@@ -15,11 +15,13 @@ buildscript {
     ext.detekt_gradle_plugin_version = "1.13.1"
     ext.bintray_gradle_plugin_version = "1.8.5"
     ext.spotbugs_gradle_plugin_version = "4.5.1"
+    ext.dokka_version = "1.4.20"
 
     repositories {
         google()
         jcenter()
         maven { url "https://plugins.gradle.org/m2/" }
+        maven { url "https://dl.bintray.com/kotlin/dokka/" }
```

Comment:
Bintray? Is this their main repo?

Reply:
They moved to Maven Central already, this is probably not needed.
https://github.com/Kotlin/dokka/issues/1725#issuecomment-773162475